storybook/src/components/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { TextArea, TextAreaProps } from '@amsterdam/design-system-react'
+import { useArgs } from '@storybook/preview-api'
+import { Meta, StoryObj } from '@storybook/react'
+import { exampleParagraph } from '../shared/exampleContent'
+
+const paragraph = exampleParagraph()
+
+type StoryProps = TextAreaProps & { invalid?: boolean }
+
+const meta = {
+  title: 'Components/Forms/Text Area',
+  component: TextArea,
+  args: {
+    defaultValue: paragraph,
+    disabled: false,
+    name: 'text-area',
```

Comment:
What is this for?

Reply:
Otherwise it generates an error in the browser console.

Reply:
You mean an issue? A `name` is not strictly necessary, and it's a bit confusing here imo.

Reply:
<img width="513" alt="image" src="https://github.com/Amsterdam/design-system/assets/474946/f4b3014a-0319-40b5-bc1e-da0e2e793909">

Yes an issue, I've removed it.